WPMedia/fusion-news-theme-blocks
# HOTFIX - TMEDIA-214 - Add filter item for 16:9 image ratio

## Description
Add missing resized image param values for 16:9 image ratio that's causing extra large top table list image to not render when selecting 16:9 ratio

## Jira Ticket
- [TMEDIA-214](https://arcpublishing.atlassian.net/browse/TMEDIA-214)


## Test Steps
- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout HOTFIX-TMEDIA-214-16-9-top-table-list`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/top-table-list-block`
3. Add a top table list block to a page and verify the extra large story can have image ratio of 16:9

## Effect Of Changes

<img width="1251" alt="TMEDIA-214-fix" src="https://user-images.githubusercontent.com/868127/115888772-34301180-a44b-11eb-8204-2fe308cbeb81.png">



## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
